website/snippets/_model-version-callout.md

```diff
@@ -0,0 +1,8 @@
+
+:::info Model versions, dbt_project.yml versions, and .yml versions
```

Comment:
changed this callout from 'version-callout' to the more appropriate 'model-version-callout'.